src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -6999,6 +6999,23 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
                 tmp = ASR::make_LogicalConstant_t(al, loc, is_ascii,
                         ASRUtils::TYPE(ASR::make_Logical_t(al, loc, 4)));
                 return;
+            } else if (attr_name == "isspace") {
+                /*
+                  * Specification:
+                  Return true if all characters in the input string are considered whitespace characters,
+                  as defined by CPython. Return false otherwise. For now we use the std::isspace function, 
+                  but if we later discover that it differs from CPython, we will have to use something else.
```

Comment:
It's still longer than 80 characters.
```suggestion
Return true if all characters in the input string are considered whitespace
characters, as defined by CPython. Return false otherwise. For now we use the
std::isspace function, but if we later discover that it differs from CPython,
we will have to use something else.
```

Reply:
Sorry, I will configure my vscode to add a vertical line after 72 char.

Reply:
Use 80, that's what I use.